docs/howto/use-extensions/list-extensions.rst

```diff
@@ -0,0 +1,11 @@
+.. _list-extensions:
+
+List extensions
+===============
+
+Different versions of Snapcraft and its various cores support different extensions. To
+list all extensions supported by the installed version of Snapcraft, run:
```

Comment:
Is it worth noting that this command doesn't tell you whether an extension is experimental or not?

Reply:
Yes! [f296e3f](https://github.com/canonical/snapcraft/pull/5212/commits/f296e3f49f5fbce6c586eb21a378d48c78f80af3)